server/tests/order/test_service.py

```diff
@@ -1257,3 +1259,144 @@ async def test_stripe_order_not_pending(
         enqueue_job_mock.assert_called_once_with(
             "order.balance", order_id=order.id, charge_id="stripe_payment_123"
         )
+
+
+@pytest.mark.asyncio
+class TestOrderServiceDunning:
```

Comment:
I usually name test classes after the method I'm testing: `TestHandlePaymentFailure`. The idea is to group all the tests of a method inside a class, just to keep things (a bit) organised.

Reply:
Upss, right! I will modify that

Reply:
DOne